test/tests/inittest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This have to be first test for cockpit
+#  install build dependencies
+#  download new version of cockpit from git 
+#  compile it
+# it is basic test compilation must PASS
+
+set -x
+
+BUILD="build1"
+PACKAGE="cockpit"
+SOURCE="https://github.com/$PACKAGE-project/$PACKAGE.git"
+
+#
+yum-builddep -y $PACKAGE
```

Comment:
> Is it able to find actual dependencies ?

You can use the `tools/cockpit.spec` file, which we keep up-to-date for the existing integration tests.

```
yum-builddep tools/cockpit.spec
```